proto/v3/message_contents/association.proto

```diff
@@ -6,18 +6,35 @@ package xmtp.v3.message_contents;
 option go_package = "github.com/xmtp/proto/v3/go/v3/message_contents";
 option java_package = "org.xmtp.proto.v3.message.contents";
 
-// Allows for us to update the format of the association text without
-// incrementing the entire proto
-enum AssociationTextVersion {
-    ASSOCIATION_TEXT_VERSION_UNSPECIFIED = 0;
-    ASSOCIATION_TEXT_VERSION_1 = 1;
-}
-
 // EIP191Association is used for all EIP 191 compliant wallet signatures
 message Eip191Association {
-    AssociationTextVersion association_text_version = 1;
+    reserved 1;
```

Comment:
Do we care about reserving fields if we aren't yet worried about interoperability between existing versions. Think we can just throw away all our existing v3 keys and start fresh